web/index.html

```diff
@@ -104,14 +107,14 @@
           name: "terminal",
           height: 550,
           prompt: "[[;;;prompt]&#8811; ]",
-          clear: false, // do not include 'clear' command
-          exit: false, // do not include 'exit' command
+          // clear: false, // do not include 'clear' command
+          // exit: false, // do not include 'exit' command
           checkArity: false,
           historySize: 200,
-          historyFilter: function(line) {
+          historyFilter(line) {
             return line.trim() !== "";
           },
-          completion: function(inp, cb) {
+          completion(inp, cb) {
```

Comment:
how... does that even work? :smile: 

What is the advantage over the old `completion: function(…) { … }`? Is this supported in most browsers?

Reply:
It's basically just a shorter version and got added to object literals in ES6. Therefore it is supported every browser except IE. Changing it was primarily just a habit

Reply:
Ok - thanks